src/init.cpp

```diff
@@ -1203,6 +1203,7 @@ bool AppInitParameterInteraction()
         return InitError("unknown rpcserialversion requested.");
 
     nMaxTipAge = gArgs.GetArg("-maxtipage", DEFAULT_MAX_TIP_AGE);
+    nMaxTipAge = std::min(nMaxTipAge, DEFAULT_MAX_TIP_AGE);
```

Comment:
Do we still limit the value to 10 minutes?

Reply:
We can remove it.